src/plugins/transport_scene_manager/TransportSceneManager.hh

```diff
@@ -15,23 +15,23 @@
  *
 */
 
-#ifndef IGNITION_GUI_PLUGINS_TRANSPORTSCENEMANAGER_HH_
-#define IGNITION_GUI_PLUGINS_TRANSPORTSCENEMANAGER_HH_
+#ifndef GZ_GUI_PLUGINS_TRANSPORTSCENEMANAGER_HH_
+#define GZ_GUI_PLUGINS_TRANSPORTSCENEMANAGER_HH_
 
 #include <memory>
 
-#include "ignition/gui/Plugin.hh"
+#include "gz/gui/Plugin.hh"
 
-namespace ignition
+namespace gz
 {
 namespace gui
 {
 namespace plugins
 {
   class TransportSceneManagerPrivate;
 
-  /// \brief Provides a Gazebo Transport interface to
-  /// `ignition::gui::plugins::MinimalScene`.
+  /// \brief Provides an Gazebo Transport interface to
```

Comment:
🤔 🧐

Reply:
:face_exhaling: 
Grammar :(

I spotted a couple of these in downstream packages. I'll push fixes :x